chutney/environment/src/main/java/com/chutneytesting/environment/domain/eventEmitter/EnvironmentEventPublisher.java

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright 2017-2023 Enedis
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.chutneytesting.environment.domain.eventEmitter;
+
+public interface EnvironmentEventPublisher {
```

Comment:
To be discussed, today we have server-core for common interface and model. We could add an interface to do that.

Reply:
So, what do you want to do ?